Mathlib/Data/List/DropRight.lean

```diff
@@ -247,4 +247,22 @@ theorem rtakeWhile_idempotent : rtakeWhile p (rtakeWhile p l) = rtakeWhile p l :
   rtakeWhile_eq_self_iff.mpr fun _ => mem_rtakeWhile_imp
 #align list.rtake_while_idempotent List.rtakeWhile_idempotent
 
+
+lemma rdrop_add {l : List α} (i j : ℕ) :
+    l.rdrop (i + j) = (l.rdrop i).rdrop j := by
+  simp_rw [List.rdrop_eq_reverse_drop_reverse, reverse_reverse,
+           drop_drop, Nat.add_comm]
+
+@[simp]
+lemma rdrop_append_length {l₁ l₂ : List α} :
+    List.rdrop (l₁ ++ l₂) (List.length l₂) = l₁:= by
+  rw [List.rdrop_eq_reverse_drop_reverse, ← length_reverse l₂,
+      reverse_append, drop_left, reverse_reverse]
+
+@[simp]
+lemma rdrop_append {l₁ l₂ : List α} (i : ℕ) :
```

Comment:
`rdrop_append_length_add`? Also, could you please add lemma `rdrop_append_of_le_length` saying `k ≤ length l₂ → List.rdrop (l₁ ++ l₂) k = l₁ ++ List.rdrop l₂ k`?

Reply:
Added; my proof can certainly be golfed further.